hrm-domain/hrm-core/case/sql/caseSearchSql.ts

```diff
@@ -373,3 +373,18 @@ export const selectCaseSearchByProfileId = selectSearchCaseBaseQuery(
     SELECT "caseId" FROM "Contacts" "c" WHERE "c"."profileId" = $<profileId> AND "c"."accountSid" = $<accountSid>
   )`,
 );
+
+// export const selectCasesByIds = (
+//   viewPermissions: TKConditionsSets<'case'>,
+//   userIsSupervisor: boolean,
+// ): SearchQueryBuilder => {
+//   return selectSearchCaseBaseQuery(
+//     `WHERE cases."accountSid" = $<accountSid> AND ${listCasesPermissionWhereClause(
+//       viewPermissions as CaseListCondition[][],
+//       userIsSupervisor,
+//     )} AND cases."id" = ANY($<caseIds::INTEGER>)`,
+//   );
+// };
+export const selectCasesByIds = selectSearchCaseBaseQuery(
+  `WHERE cases."accountSid" = $<accountSid> AND cases."id" = ANY($<caseIds>::INTEGER[])`,
+);
```

Comment:
A few notes here:
- If the commented out code is not needed, can we get rid of it?
- How is this handling the "filter by permissions" part? I thought that was what `AND ${listCasesPermissionWhereClause( viewPermissions as CaseListCondition[][], userIsSupervisor, )}` portion of the query is supposed to do?
  - I know I've said let's skip adding tests in order to stick to our schedule, but manually testing this scenarios would be great :p

Reply:
@mythilytm I'm sorry, it looks like this feedback was not required since `selectSearchCaseBaseQuery` function already adds the correct `listCasesPermissionWhereClause` permission based filters. Adding the extra params caused an error type. I removed  the redudant clause and it should be fine now, while still preserving the intended behavior.